src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -708,7 +716,6 @@
     </COMReference>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Condition="$(BuildForRelease) == 'true'" Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkGitHubVersion)" PrivateAssets="All" />
```

Comment:
For some reason, I could not find this in the previous commit or where it was removed from.

Reply:
The purpose of `$(BuildForRelease) == 'true'` condition is to avoid including this package if the condition is not correct.
Possibly, you chose an incorrect location for this change.

Reply:
I tried without `$(BuildForRelease) == ''` and didn't see any issue with the `msbuild` run.

Reply:
You mentioned "Release Build properties must be turned on for Release purposes and turned off for Code Coverage calculations. Condition="$(BuildForRelease) == 'true'" needs to be moved to the Version.props as well."

Let me explain the issue once more.  With Condition="$(BuildForRelease) == 'true'" inside Versions.props, the nupkg file gets created but when opened in Nuget package explorer, the Source Link information will show "Invalid link" error. This is because the Condition $(BuildForRelease) is empty or not initialized yet inside Versions.props. Thus, the package Microsoft.SourceLink.GitHub did not get included in the build and used in the nupkg file.

Therefore, another condition is Or'ed with it:  Condition="$(BuildForRelease) == '' " to ensure that the Microsoft.SourceLink.GitHub package gets included in the build and eventually used in the nupkg file.

Lastly, in build,proj, you'll notice that $(BuildForRelease) is initialized to 'true' if it is empty much later that when the Directory.Build.props is imported.

Reply:
In light of the pipeline error message about not able to build test projects because of missing reference to Microsoft.SourceLink.GitHub package, the reference to Microsoft.SourceLink.GitHub was added back to each project so they will build correctly.